src/components/sbb-form-field/sbb-form-field.stories.js

```diff
@@ -0,0 +1,438 @@
+import { h } from 'jsx-dom';
```

Comment:
This file became very large, are there any option to shorten the code by using more templates or whatever?

Reply:
Done, can you chack these?

Reply:
Some other fixes were done:
- 2 templates removed
- controls are set for each template and splitted in categories by using 'table' prop
- disabled and readonly stories are still splitted to see them in chromatic
- new ellipsis case for label and value
- fix behaviour on the error story

Reply:
It look ok for me now :)